Project.toml

```diff
@@ -11,10 +11,12 @@ DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
 ExponentialUtilities = "d4d017d3-3776-5f7e-afef-a10c40355c18"
 GR = "28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+LsqFit = "2fda8390-95c7-5789-9bda-21331edee243"
```

Comment:
Did you end up using `LsqFit.jl`?

Reply:
No, I used bboptim. Will remove it right away.